### PR TITLE
Up coinbase lock height to 2 hours

### PR DESCRIPTION
**Pull request by SWvheerden**
## Description

This increases the coinbase lock height to two hours up from the current 1 block. 
This will be effective from block 10100.

## Motivation and Context

We need to start testing coinmbase lockheights. 

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [x] I ran `cargo test` successfully before submitting my PR.
* [ ] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: base_layer/core/src/consensus/consensus_constants.rs [hunk 1]
```diff
@@ -358,21 +358,38 @@ impl ConsensusConstants {
...
+                median_timestamp_count: 11,
+                emission_initial: 5_538_846_115 * uT,
+                emission_decay: &EMISSION_DECAY,
+                emission_tail: 100.into(),
+                max_randomx_seed_height: std::u64::MAX,
+                proof_of_work: algos.clone(),
+                faucet_value: (5000 * 4000) * T,
+            },
+            ConsensusConstants {
+                effective_from_height: 0,
```
Comment (stringhandler) [inline base_layer/core/src/consensus/consensus_constants.rs]: This needs a value

Comment (SWvheerden) [reply base_layer/core/src/consensus/consensus_constants.rs]: thanks. I see I updated the wrong one, updated mainnet...

**Review by stringhandler — APPROVED** [review]
Looks good, but will only merge it just before a release otherwise will cause a chain split

**Comment by stringhandler** [comment]
Wouldn't an hour be 30 blocks?

**Comment by SWvheerden** [comment]
> Wouldn't an hour be 30 blocks?

Yip, this is 2 hours

**Comment by SWvheerden** [comment]
Closing in favour of #2555

**Pull request closed without merging**